chaco/base_xy_plot.py

```diff
@@ -75,7 +75,7 @@ class BaseXYPlot(AbstractPlotRenderer):
     orientation = Enum("h", "v")
 
     #: Overall alpha value of the image. Ranges from 0.0 for transparent to 1.0
-    alpha = Range(0.0, 1.0, 1.0)
+    alpha = Range(0.0, 1.0, 1.0, requires_redraw=True)
```

Comment:
This will now spill over into other classes which inherit from `BaseXYPlot` i.e. any subclass of `BaseXYPlot` will invalidate and redraw if the alpha value changes.. That might be the right behavior but we need to be conscious of the fact that we might be modifying behavior.